builtin/logical/ssh/path_config_ca.go

```diff
@@ -228,6 +228,17 @@ func (b *backend) pathConfigCAUpdate(req *logical.Request, data *framework.Field
 
 		return nil, err
 	}
+
+	if generateSigningKey {
+		response := &logical.Response{
+			Data: map[string]interface{}{
```

Comment:
@jefferai I agree, a read operation on the config/ca path would be very useful and logical.